hangman.js

```diff
@@ -0,0 +1,139 @@
+var inquirer = require("inquirer");
+var Word = require("./word.js");
+
+// Prearranged words for the hangman program
+var words = ['PROGRAM', 'LANGUAGE', 'STRING', 'JAVASCRIPT', 'PYTHON'];
+
+// Array to store word Objects that are created based on the words above
+var wordArray = [];
+
+
+// Function to initalize wordArray with word Objects
+function initWords() {
+	for (var i=0; i<words.length; i++) {
+		var newWord = new Word(words[i]);
+		wordArray.push(newWord);
+	}
+}
+
+// Function to reset all the words for a fresh start of the game
+function resetGame() {
+	console.log("RESET game!");
+	for (var i=0; i < wordArray.length; i++) {
+		wordArray[i].resetWord();
+	}
+}
+
+// Represents the round we are playing. We start with round 0.
+// If all words are finished, user gets a choice to re play the game or end it.
+var round = 0;
+
+// Main function that has the logic for Hangman Game
+function playGame() {
+	console.log("ROUND: " + round);
+	var display_str = wordArray[round].displayWord();
+	console.log(display_str);
+	inquirer.prompt([
+		{
+			name: "guess",
+			message: "Guess a letter? "
```

Comment:
This should just be a statement since you aren't asking if the user can guess the letter.